common/static/common/js/spec/components/tabbed_view_spec.js

```diff
@@ -7,7 +7,16 @@
             'common/js/components/views/tabbed_view'
            ],
            function($, _, Backbone, TabbedView) {
-               var view,
+               var keys = {
+                       'UP': 38,
+                       'DOWN': 40,
+                       'LEFT': 37,
+                       'RIGHT': 39
+                   },
+                   keyPressEvent = function(key) {
+                       return $.Event('keydown', { keyCode: key });
```

Comment:
Our current best practice is to use the JQuery.simulate library for key presses. @AlasdairSwan might be able to suggest a better approach too.